aws/resource_aws_instance_test.go

```diff
@@ -2719,3 +2821,21 @@ resource "aws_instance" "foo" {
   }
 }`, rInt, rInt)
 }
+
+func testAccInstanceConfig_getPasswordData(val bool) string {
+	return fmt.Sprintf(`
+	resource "aws_key_pair" "foo" {
+		key_name = "tf-acc-winpasswordtest"
```

Comment:
The key name needs to be randomized for parallel or broken test runs. `key_name = "tf-acc-%d"` (where `%d` can come from the top level test function, e.g. `rInt := acctest.RandInt()`